plugins/languages/rust/rust-tools.nix

```diff
@@ -58,26 +58,44 @@ in
     };
 
     hoverActions = {
-      border = helpers.defaultNullOpts.mkBorder ''
+      border = helpers.defaultNullOpts.mkBorder [
         [
-          [ "╭" "FloatBorder" ]
-          [ "─" "FloatBorder" ]
-          [ "╮" "FloatBorder" ]
-          [ "│" "FloatBorder" ]
-          [ "╯" "FloatBorder" ]
-          [ "─" "FloatBorder" ]
-          [ "╰" "FloatBorder" ]
-          [ "│" "FloatBorder" ]
+          "╭"
+          "FloatBorder"
         ]
-      '' "rust-tools hover window" "";
+        [
+          "─"
+          "FloatBorder"
+        ]
+        [
+          "╮"
+          "FloatBorder"
+        ]
+        [
+          "│"
+          "FloatBorder"
+        ]
+        [
+          "╯"
+          "FloatBorder"
+        ]
+        [
+          "─"
+          "FloatBorder"
+        ]
+        [
+          "╰"
+          "FloatBorder"
+        ]
+        [
+          "│"
+          "FloatBorder"
+        ]
+      ] "rust-tools hover window" "";
 
-      maxWidth =
-        helpers.defaultNullOpts.mkNullable types.int null
-          "Maximal width of the hover window. null means no max.";
+      maxWidth = helpers.defaultNullOpts.mkInt null "Maximal width of the hover window. null means no max.";
 
-      maxHeight =
-        helpers.defaultNullOpts.mkNullable types.int null
-          "Maximal height of the hover window. null means no max.";
+      maxHeight = helpers.defaultNullOpts.mkInt null "Maximal height of the hover window. null means no max.";
```

Comment:
```suggestion
      maxHeight = helpers.defaultNullOpts.mkInt null ''
        Maximal height of the hover window. null means no max.
      '';
```

Reply:
This was only moved onto one line because we no longer hit the max line length (`mkInt` is shorter than `mkNullable types.int`).

I know we usually use indented strings, but this file predominantly uses ordinary string literals and I was trying to minimize changes.